src/config/__tests__/settings.spec.ts

```diff
@@ -0,0 +1,228 @@
+import fs from 'fs';
+
+import * as settings from '../settings';
+import { PathManagementStrategy } from '@/integrations/pathManager';
+import { RecursivePartial } from '~/utils/typeUtils';
+
+describe('updateFromCommandLine', () => {
+  let prefs: settings.Settings;
+  let configDir: string;
+
+  beforeEach(() => {
+    jest.spyOn(fs, 'writeFileSync').mockImplementation(() => { });
+    prefs = {
+      version:    4,
+      kubernetes: {
+        version:                    '1.23.5',
+        memoryInGB:                 4,
+        numberCPUs:                 2,
+        port:                       6443,
+        containerEngine:            settings.ContainerEngine.MOBY,
+        checkForExistingKimBuilder: false,
+        enabled:                    true,
+        WSLIntegrations:            {},
+        options:                    {
+          traefik: true,
+          flannel: false,
+        },
+        suppressSudo: false
+      },
+      portForwarding: { includeKubernetesServices: false },
+      images:         {
+        showAll:   true,
+        namespace: 'k8s.io'
+      },
+      telemetry:              true,
+      updater:                true,
+      debug:                  true,
+      pathManagementStrategy: PathManagementStrategy.NotSet,
+    };
+  });
+
+  describe('getUpdatableNode', () => {
+    test('returns nil on an invalid top level accessor', () => {
```

Comment:
```suggestion
    test('returns null on an invalid top level accessor', () => {
```
JS doesn't have `nil` :)

Reply:
sure